### DISCUSSION TITLE
Usage of ENUM_SENS_SMOOTHING

### DISCUSSION THREAD

**Discussion by thomasdick** [su2code/SU2#1448] [Q&A] [answered]
Hi,

I found the following code in the master branch in file Common/include/option_structure.hpp:1937

/*!
 * \brief Types of sensitivity smoothing
 */
enum ENUM_SENS_SMOOTHING {
  NO_SMOOTH = 0,  /*!< \brief No smoothing. */
  SOBOLEV = 1,    /*!< \brief Sobolev gradient smoothing. */
  BIGRID = 2      /*!< \brief Bi-grid technique smoothing. */
};
static const MapType<std::string, ENUM_SENS_SMOOTHING> Sens_Smoothing_Map = {
  MakePair("NONE", NO_SMOOTH)
  MakePair("SOBOLEV", SOBOLEV)
  MakePair("BIGRID", BIGRID)
};

I cannot find a place where this is used? 
Since it seems related to sensitivity smoothing and I'm currently working on this, I'm interested if this is in use somewhere?

**Comment by pcarruscag** [comment, marked as answer]
As far as I know it is not, I think I kept it around from remembering you were working on sensitivity smoothing.

**Comment by thomasdick** [comment]
Thank you for the info Pedro.
I will check if I can use it in my draft.
